.github/dependabot.yaml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  package-ecosystem: npm
```

Comment:
This is supposed to be a list, isn't it?

Reply:
Yep, done!